test/functional/feature_asmap.py

```diff
@@ -75,7 +75,7 @@ def test_default_asmap(self):
     def test_default_asmap_with_missing_file(self):
         self.log.info('Test dashd -asmap with missing default map file')
         self.stop_node(0)
-        msg = "Error: Could not find or parse specified asmap: '\"{}\"'".format(self.default_asmap)
+        msg = "Error: Could not find asmap file '\"{}\"'".format(self.default_asmap)
         self.node.assert_start_raises_init_error(extra_args=['-asmap'], expected_msg=msg)
 
     def run_test(self):
```

Comment:
missing changes https://github.com/bitcoin/bitcoin/commit/b8d0412b213df18f23bf8677ab94068c6cca9f51#diff-b1e19192258d83199d8adaa5ac31f067af98f63554bfdd679bd8e8073815e69dR1836-R1845

Reply:
Already present in https://github.com/dashpay/dash/pull/4028/files#diff-b1e19192258d83199d8adaa5ac31f067af98f63554bfdd679bd8e8073815e69dR1942-R1950